changelogs/v0.25.0.md

```diff
@@ -0,0 +1,131 @@
+* **authenticate**: verify redirect in Callback test by @kenjenkins in https://github.com/pomerium/pomerium/pull/4894
+* **config**: remove unnecessary authenticate route when using hosted authenticate (authenticate.pomerium.app) by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4719
+* **config**: Add a global config option for pass_identity_headers, in addition to existing per-route option by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4720
+* **config**: disable strict-transport-security header with staging autocert by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4741
+* **config**: no longer stub out HPKE public key fetch by @kenjenkins in https://github.com/pomerium/pomerium/pull/4853
+* **runtime**: update to Go 1.21.4 by @kenjenkins in https://github.com/pomerium/pomerium/pull/4770
+* **runtime**: automatically determine goroutine max cap by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4766
+* **session**: add unit tests for gRPC wrapper methods by @kenjenkins in https://github.com/pomerium/pomerium/pull/4713
+* **tests**: renew test certs by @kenjenkins in https://github.com/pomerium/pomerium/pull/4738
+* **tests**: add tool for renewing test certs by @kenjenkins in https://github.com/pomerium/pomerium/pull/4742
+* **tests**: re-generate test configurations by @kenjenkins in https://github.com/pomerium/pomerium/pull/4816
+* **tests**: check for profile cookies by @kenjenkins in https://github.com/pomerium/pomerium/pull/4847
+* **zero**: rebase and merge feature/zero branch by @kenjenkins in https://github.com/pomerium/pomerium/pull/4745
+* **zero**: fix restart behavior by @kenjenkins in https://github.com/pomerium/pomerium/pull/4753
+* **zero**: use os.UserCacheDir for boostrap config path by @kenjenkins in https://github.com/pomerium/pomerium/pull/4744
+* **zero**: better code reuse by @wasaga in https://github.com/pomerium/pomerium/pull/4758
+* **zero**: set drwx------ for cache dir by @wasaga in https://github.com/pomerium/pomerium/pull/4764
+* **zero**: support gzipped blobs by @wasaga in https://github.com/pomerium/pomerium/pull/4767
+* **zero**: add linear probabilistic counter for MAU estimation by @wasaga in https://github.com/pomerium/pomerium/pull/4776
+* **zero**: use production urls by default by @wasaga in https://github.com/pomerium/pomerium/pull/4814
+* **zero**: add more verbose logging about background control loops by @wasaga in https://github.com/pomerium/pomerium/pull/4815
+* **zero**: calculate DAU and MAU by @wasaga in https://github.com/pomerium/pomerium/pull/4810
+* **zero**: add reporter by @wasaga in https://github.com/pomerium/pomerium/pull/4855
+* **zero**: add support for managed mode from config file by @calebdoxsey in https://github.com/pomerium/pomerium/pull/4756
+
+### Dependency Updates
+* bump github.com/go-jose/go-jose/v3 from 3.0.0 to 3.0.1 by @dependabot in https://github.com/pomerium/pomerium/pull/4760
```

Comment:
I wonder if we especially want to call out the two dependabot security updates; maybe something inline like this:

```suggestion
* bump github.com/go-jose/go-jose/v3 from 3.0.0 to 3.0.1 by @dependabot in https://github.com/pomerium/pomerium/pull/4760. This includes a patch for [GHSA-2c7c-3mj9-8fqh](https://github.com/advisories/GHSA-2c7c-3mj9-8fqh). It is unclear whether this vulnerability could be exploited to cause a denial of service issue in Pomerium.
```

(or else maybe a separate 'Security' heading? although neither has a clear security impact on Pomerium itself)